hub/core/compute/process.py

```diff
@@ -8,14 +8,14 @@ class ProcessProvider(ComputeProvider):
     def __init__(self, workers):
         self.workers = workers
         self.pool = ProcessPool(nodes=workers)
-        self.manager = pathos_multiprocess.Manager()
+        self._manager = pathos_multiprocess.Manager()
         self._closed = False
 
     def map(self, func, iterable):
         return self.pool.map(func, iterable)
 
-    def create_queue(self):
-        return self.manager.Queue()
+    def manager(self):
```

Comment:
```suggestion
manager looks like it's a property
```

Reply:
This probably will change. Ray doesn't have manager and variable communication would have to be done using some other mehtod.